docs/tasks/sync_broker.md

```diff
@@ -0,0 +1,15 @@
+---
+title: Sync a Broker
+layout: docwithnav
+---
+
+A broker's catalog may occasionaly change the services it offers. A broker
```

Comment:
occasionally

Reply:
Therefore, Service Catalog must resync with the broker to get the updated services

Reply:
add the name of the global flag in the helm chart